functions/exercise2/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
```

Comment:
I do not have appropriate place to report it, so I will do it here - what is the purpose of adding empty style.css file in this pull request?